runtimes/README.md

```diff
@@ -223,6 +223,19 @@ The Identity Management feature is designed to centralize the management of auth
 
 Complete Identity Management parameter and result interfaces can be found in [identity-management.ts](src/protocol/identity-management.ts)
 
+### Notification
+
+The notification feature can be used to send custom customer-facing notifications to clients. Notifications can contain actions, like show URL, but also followup actions, like request customer acknowledgement. When customer reacts to followup actions, asyncronous notification can be sent from client to server to notify server about this.
```

Comment:
nit: type asyncronous -> asynchronous

Would it be helpful to compare/contrast with the built-in LSP ShowMessage and/or give rationale for when to use which one?  

https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_showMessage

Reply:
Will do in followup PR.